astrowidgets/ginga.py

```diff
@@ -973,12 +974,23 @@ def scroll_pan(self, value):
         else:
             bindmap.map_event(None, (), 'pa_pan', 'zoom')
 
-    def save(self, filename):
+    def save(self, filename, overwrite=False):
         """
         Save out the current image view to given PNG filename.
+
+        Parameters
+        ----------
+        filename : str
```

Comment:
Should this be str or Path too?

Reply:
Heck, probably should be both. Will fix in a separate PR since I was a little too quick on the merge button this morning